crates/subspace-service/src/chain_spec.rs

```diff
@@ -55,8 +54,8 @@ const TOKEN_GRANTS: &[(&str, u128)] = &[
     ("5FZwEgsvZz1vpeH7UsskmNmTpbfXvAcojjgVfShgbRqgC1nx", 27_800),
 ];
 
-/// Specialized `ChainSpec`. This is a specialization of the general Substrate ChainSpec type.
-pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig>;
+/// The `ChainSpec` parameterized for the subspace runtime.
+pub type SubspaceChainSpec = sc_service::GenericChainSpec<subspace_runtime::GenesisConfig>;
```

Comment:
Why are all these things are now prefixed with `Subspace` or `subspace_`?

Reply:
Useful when you have multiple runtimes here, take polkadot as an example, they have Polkadot, Kusama, Westend, etc. Even there is only one runtime for now, it does not hurt to have a specific prefix.